<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InverterWindow</class>
 <widget class="QWidget" name="InverterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>352</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>2000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Bidirectional Inverter System</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res.qrc">
    <normaloff>:/resources/icons/solar-inverter-crop.png</normaloff>:/resources/icons/solar-inverter-crop.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Parameter</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_2" columnstretch="6,12,1,0">
              <item row="6" column="2">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>kW</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QSpinBox" name="invertervoltagecutoffspinBox_2">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="maximum">
                 <number>1500000</number>
                </property>
                <property name="value">
                 <number>768000</number>
                </property>
               </widget>
              </item>
              <item row="3" column="2">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>mV</string>
                </property>
               </widget>
              </item>
              <item row="4" column="2">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>mV</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="invertermodecomboBox">
                <item>
                 <property name="text">
                  <string>Charge</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Discharge</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Cycling</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Voltage LL Cut Off</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QSpinBox" name="invertervoltagecutoffspinBox_1">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="maximum">
                 <number>1000000</number>
                </property>
                <property name="value">
                 <number>793600</number>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Current</string>
                </property>
               </widget>
              </item>
              <item row="5" column="2">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>mA</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Power</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Mode</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>mV</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>Voltage</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QSpinBox" name="invertercurrentspinBox">
                <property name="maximum">
                 <number>120000</number>
                </property>
                <property name="value">
                 <number>5000</number>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Voltage L Cut Off</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QSpinBox" name="inverterchargevoltagespinBox">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="maximum">
                 <number>1000000</number>
                </property>
                <property name="value">
                 <number>921600</number>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="invertercontroltypecomboBox">
                <item>
                 <property name="text">
                  <string>Constant Current</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Constant Power</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Control Mode</string>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QDoubleSpinBox" name="inverterpowerdoubleSpinBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="maximum">
                 <double>1000000.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="3" column="3">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="startrinverterpushButton">
                <property name="text">
                 <string>Start</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="stopinverterpushButton">
                <property name="text">
                 <string>Stop</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Bidirectional Inverter System</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
